src/core/lib/attribute-transition-manager.js

```diff
@@ -173,20 +173,9 @@ export default class AttributeTransitionManager {
   _runTransformFeedback({uniforms, buffers}) {
     const {model, transformFeedback} = this;
 
-    // Cannot bind to transform feedback if it's already used as attribute
-    Object.values(buffers).forEach(buffer => buffer.unbind());
+    transformFeedback.bindBuffers(buffers, {});
 
-    model.program.use();
-    transformFeedback.bindBuffers(buffers, {clear: true});
-    transformFeedback.begin(GL.POINTS);
-
-    const parameters = {
-      [GL.RASTERIZER_DISCARD]: true
-    };
-
-    model.draw({uniforms, parameters});
-
-    transformFeedback.end();
+    model.draw({uniforms, transformFeedback});
```

Comment:
As I commented in luma PR, to avoid confusion, parameters with `[GL.RASTERIZER_DISCARD]: true` should still need to sent. Or lets wait until that PR is finalized.